contract/bridge/contracts/Collateral.sol

```diff
@@ -47,6 +47,10 @@ abstract contract ICollateral {
         emit SlashCollateral(from, to, amount);
     }
 
+    function get_free_collateral(address vault_id) internal view returns(uint256) {
```

Comment:
@peekpi can u also follow the coding guidelines? names using mixedCase? thanks.

Reply:
@peekpi let's also add NatSpec formatting for better readability. e.g.

```/**
    * @notice Checks if the given period has expired since the given starting point. This calculation is based on the activeBlockCount.
    * @param opentime the activeBlockCount at the time the issue/redeem/replace was opened.
    * @param period the number of blocks the user or vault has to complete the action.
    * @return true if the period has expired.
    */```